cl/corpus_importer/management/commands/make_aws_manifest_files.py

```diff
@@ -187,6 +210,341 @@ def get_custom_query(
     return query, params
 
 
+def upload_manifest(
+    record_ids: list[tuple[int]],
+    base_filename: str,
+    options: dict[str, Any],
+    batch_counter: int | None = None,
```

Comment:
It seems that these parameter names are outdated in the docstring.

Reply:
You're right! Thanks!